test/e2e/pod_create_test.go

```diff
@@ -304,7 +304,7 @@ var _ = Describe("Podman pod create", func() {
 		// Make sure we error out with --name.
 		session := podmanTest.Podman([]string{"pod", "create", "--replace", ALPINE, "/bin/sh"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(125))
+		Expect(session).Should(ExitWithError(125, "accepts at most 1 arg(s), received 2"))
```

Comment:
*Eyeballs please*: I sort of think this is not the actual intention of the test, but I can't figure out what it's supposed to test.

Reply:
I think the intent was to require a name be passed when replacing. Removing `ALPINE, "/bin/sh"` would do that.

Reply:
Oh, duh, `cannot replace pod without --name being set`. I thought I had tried that; obviously not. Thank you!